contracts/mocks/JBFundingCycleDataSourceMock.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import './../interfaces/IJBFundingCycleDataSource.sol';
+
+contract JBFundingCycleDataSourceMock is IJBFundingCycleDataSource {
+  function payParams(JBPayParamsData calldata _param)
+    external
+    view
+    override
+    returns (
+      uint256 weight,
+      string memory memo,
+      IJBPayDelegate delegate,
+      bytes memory delegateMetadata
+    )
+  {}
+
+  function redeemParams(JBRedeemParamsData calldata _param)
+    external
+    view
+    override
+    returns (
+      uint256 amount,
+      string memory memo,
+      IJBRedemptionDelegate delegate,
+      bytes memory delegateMetadata
+    )
+  {}
+}
```

Comment:
As mentioned over Discord, we should need to create mock contracts for interfaces. You can call `deployMockContract` against the interface ABI. Let's remove these. JBProject was different since it's not an interface and is never extended by any of the core JB contracts. The intent is for projects to extend this themselves.